src/elements/button/mini-button-group/mini-button-group.stories.ts

```diff
@@ -0,0 +1,108 @@
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { InputType } from '@storybook/types';
+import type {
+  Args,
+  ArgTypes,
+  Decorator,
+  Meta,
+  StoryContext,
+  StoryObj,
+} from '@storybook/web-components';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+
+import { sbbSpread } from '../../../storybook/helpers/spread.js';
+
+import readme from './readme.md?raw';
+import './mini-button-group.js';
+import '../mini-button.js';
+import '../../divider/divider.js';
+
+const size: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['s', 'm', 'l', 'xl'],
+};
+
+const negative: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const accessibilityLabel: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const defaultArgTypes: ArgTypes = {
+  size,
+  negative,
+  'accessibility-label': accessibilityLabel,
+};
+
+const defaultArgs: Args = {
+  size: size.options![1],
+  negative: false,
+  'accessibility-label': '',
+};
+
+const miniButtons = (n = 1): TemplateResult[] =>
+  new Array(n)
+    .fill(null)
+    .map(() => html`<sbb-mini-button icon-name="pen-small"></sbb-mini-button>`);
```

Comment:
I wonder if we should create a more realistic like example by providing various icons instead of always the pen? it's nitpicky I know :)